src/View/CellTrait.php

```diff
@@ -109,21 +109,25 @@ protected function _createCell($className, $action, $plugin, $options)
         if (!empty($plugin)) {
             $builder->plugin($plugin);
         }
-        if (!empty($this->theme)) {
-            $builder->theme($this->theme);
-        }
         if (!empty($this->helpers)) {
             $builder->helpers($this->helpers);
             $instance->helpers = $this->helpers;
         }
-        if (isset($this->viewClass)) {
-            $builder->className($this->viewClass);
-            $instance->viewClass = $this->viewClass;
-        }
+
         if ($this instanceof View) {
+            if (!empty($this->theme)) {
+                $builder->theme($this->theme);
+            }
```

Comment:
We removed `Controller::$theme` and it's now handled with magic getter for BC, so using `$this->theme` in controller will trigger notice.

So if `$this` is not instance of `View` should we check if we can get a `ViewBuilder` instance and try to get theme from that. I will update accordingly.